scopes/lanes/lanes/lane.cmd.ts

```diff
@@ -22,14 +22,14 @@ type LaneOptions = {
 
 export class LaneListCmd implements Command {
   name = 'list';
-  description = `list lanes`;
+  description = `list local lanes`;
   alias = '';
   options = [
     ['d', 'details', 'show more details on the state of each component in each lane'],
-    ['j', 'json', 'show lanes details in a json format'],
-    ['r', 'remote <remote-scope-name>', 'show remote lanes'],
-    ['', 'merged', 'show merged lanes'],
-    ['', 'not-merged', 'show lanes that are not merged'],
+    ['j', 'json', "show lanes' details in a json format"],
+    ['r', 'remote <remote-scope-name>', 'show all remote lanes from the specified scope'],
+    ['', 'merged', 'show merged lanes'], // @david again, is this all the merged lanes in the whole org, ever?
```

Comment:
No. (I don't think we have such concept from the CLI perspective).
It's only the local lanes. Unless you pass `--remote`, in which case it's all the scope's lanes.

Reply:
ok, changed to local lanes